src/main/java/com/questhelper/quests/gertrudescat/GertrudesCat.java

```diff
@@ -0,0 +1,219 @@
+import com.questhelper.steps.conditional.Conditions;
+import com.questhelper.steps.conditional.ItemCondition;
+import com.questhelper.steps.conditional.ItemRequirementCondition;
+import com.questhelper.steps.conditional.ZoneCondition;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import net.runelite.api.IconID;
+import net.runelite.api.ItemID;
+import net.runelite.api.NpcID;
+import net.runelite.api.ObjectID;
+import net.runelite.api.coords.WorldPoint;
+
+@QuestDescriptor(
+	quest = QuestHelperQuest.GERTRUDES_CAT
+)
+public class GertrudesCat extends BasicQuestHelper
+{
+	private ItemRequirement bucketOfMilk, coins, seasonedSardine;
+
+	private QuestStep talkToGertrude, talkToChildren,
+		gertrudesCat, gertrudesCat2, searchNearbyCrates, giveKittenToFluffy,
+		finishQuest;
+
+	private ConditionForStep isUpstairsLumberyard, hasFluffsKitten;
+
+	@Override
+	public Map<Integer, QuestStep> loadSteps()
+	{
+		setupItemRequirements();
+		setupZones();
+		setupConditions();
+
+		return getSteps();
+	}
+
+	private Map<Integer, QuestStep> getSteps()
+	{
+		Map<Integer, QuestStep> steps = new HashMap<>();
+
+		steps.put(0, talkToGertrude = getTalkToGertrude());
+		steps.put(1, talkToChildren = getTalkToChildren());
+		steps.put(2, getLumberyard());
+		steps.put(3, getFeedCat());
+		steps.put(4, findFluffsKitten());
+		steps.put(5, finishQuest = returnToGertrude());
+		return steps;
+	}
+
+	private NpcStep returnToGertrude()
+	{
+		return new NpcStep(this, NpcID.GERTRUDE,
+			new WorldPoint(3148, 3413, 0), "Talk to Gertrude.");
+	}
+
+	private QuestStep findFluffsKitten()
+	{
+
+		//Need to find to ways to hide arrow
+		searchNearbyCrates = new NpcStep(this, NpcID.CRATE,
+			"Search for a kitten.", true);
+		ObjectStep climbDownLadderStep = goDownLadderStep();
+		ObjectStep climbUpLadderStep = getClimbLadder();
+		ArrayList<ItemRequirement> fluffsKittenRequirement = new ArrayList();
+		fluffsKittenRequirement.add(new ItemRequirement("Fluffs Kitten", ItemID.FLUFFS_KITTEN));
+
+		climbUpLadderStep.addItemRequirements(fluffsKittenRequirement);
+		Conditions hasFluffsKittenUpstairs = new Conditions(hasFluffsKitten, isUpstairsLumberyard);
+
+		giveKittenToFluffy = getGertrudesCat("Return the kitten to Gertrude's Cat", new ItemRequirement("Fluffs Kitten", ItemID.FLUFFS_KITTEN));
+		giveKittenToFluffy.addIcon(ItemID.FLUFFS_KITTEN);
+
+		ConditionalStep conditionalKitten = new ConditionalStep(this, searchNearbyCrates);
+		conditionalKitten.addStep(hasFluffsKittenUpstairs, giveKittenToFluffy);
+		conditionalKitten.addStep(hasFluffsKitten, climbUpLadderStep);
+		conditionalKitten.addStep(isUpstairsLumberyard, climbDownLadderStep);
+
+		searchNearbyCrates.addSubSteps(climbDownLadderStep);
+		giveKittenToFluffy.addSubSteps(climbUpLadderStep);
+
+		return conditionalKitten;
+	}
+
+	private ObjectStep goDownLadderStep()
+	{
+		return new ObjectStep(this, ObjectID.LADDER_11795, new WorldPoint(3310, 3509, 1)
```

Comment:
This step could be a bit more informative, like 'Climb down the ladder in the Lumberyard.'